Tests/UnitTests/CompatibilityTests.cpp

```diff
@@ -0,0 +1,666 @@
+#include <Babylon/AppRuntime.h>
+#include <Babylon/ScriptLoader.h>
+#include <Babylon/Polyfills/AbortController.h>
+#include <Babylon/Polyfills/Console.h>
+#include <Babylon/Polyfills/Scheduling.h>
+#include <Babylon/Polyfills/URL.h>
+#include <Babylon/Polyfills/Blob.h>
+#include <gtest/gtest.h>
+#include <future>
+#include <chrono>
+#include <tuple>
+#include <vector>
+#include <string>
+#include <utility>
+#include <atomic>
+
+namespace
+{
+    class EngineCompatTest : public ::testing::Test
+    {
+    protected:
+        Babylon::AppRuntime& Runtime()
+        {
+            if (!m_runtime)
+            {
+                m_runtime = std::make_unique<Babylon::AppRuntime>();
+                m_runtime->Dispatch([](Napi::Env env) {
+                    Babylon::Polyfills::Console::Initialize(env, [](const char*, Babylon::Polyfills::Console::LogLevel) {});
+                    Babylon::Polyfills::AbortController::Initialize(env);
+                    Babylon::Polyfills::Scheduling::Initialize(env);
+                    Babylon::Polyfills::URL::Initialize(env);
+                    Babylon::Polyfills::Blob::Initialize(env);
+                });
+
+                m_loader = std::make_unique<Babylon::ScriptLoader>(*m_runtime);
```

Comment:
I'm not sure what this is for. I think maybe it's for calling Eval? If so, we do this using `Napi::Eval`. We don't need a loader to do this. The ScriptLoader class is intended to load js files in order and optionally do js work before, after, or in between.